src/main/java/org/jenkinsci/plugins/workflow/cps/CpsBodyExecution.java

```diff
@@ -208,7 +208,7 @@ private Continuable createContinuable(CpsThread currentThread, CpsCallableInvoca
                     List<StepExecution> executions = new ArrayList<>();
                     // cf. trick in CpsFlowExecution.getCurrentExecutions(true)
                     Map<FlowHead, CpsThread> m = new LinkedHashMap<>();
-                    for (CpsThread t : g.threads.values()) {
+                    for (CpsThread t : g.getThreads()) {
```

Comment:
Note that this is now no behavioral change, you are just encapsulating the field.

Reply:
Yeah, to me it still seems valuable to encapsulate the field and only provide a read only view of the values since that is what was being used everywhere anyway, but I don't feel too strongly about it. I am happy to revert these changes to keep the diff minimal if you prefer.

Reply:
The added encapsulation seems wise.